src/api/ecosystems.ts

```diff
@@ -0,0 +1,62 @@
+import {  axiosPost, axiosPut } from "../services/apiRequests"
+import { apiRoutes } from "../config/apiRoutes";
+import { getFromLocalStorage } from "./Auth";
+import { storageKeys } from "../config/CommonConstant";
+
+export const createEcosystems = async (data: object) => {
```

Comment:
@pranalidhanavade can we not define an interface for data here instead of assigning a generic datatype as object.

Reply:
Added interface for that.